src/views/Home/index.view.tsx

```diff
@@ -10,7 +10,7 @@ const title = [
     style: "",
   },
   {
-    text: "Coming Soon",
+    text: "January 15-17, 2021",
```

Comment:
Added this date in here because we want to be [MLH compliant](https://cruzhacks2021-team.slack.com/archives/C013X5SEH37/p1604359293028800) in order to get that badge!

Reply:
^ dropped the year off the subheader since we already say it's 2021 in the title